src/query/src/promql/planner.rs

```diff
@@ -3476,6 +3501,34 @@ mod test {
         );
     }
 
+    #[tokio::test]
+    async fn test_nonexistent_label() {
```

Comment:
Need to add a sqlness test.

Reply:
resolved in [dfb98bf](https://github.com/GreptimeTeam/greptimedb/pull/5519/commits/dfb98bfeaba2dfe13ab372f9c6192f41f3490e7d)